migrate/20251209_add_limited_reputation.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    alter_table(:project) do
+      set_column_type :reputation, "text collate \"C\"", using: Sequel.cast(:reputation, String)
+      set_column_default :reputation, "new"
+      add_constraint :reputation_check, reputation: %w[new verified limited]
+    end
+    drop_enum :project_reputation
+  end
+
+  down do
+    create_enum(:project_reputation, %w[new verified])
+
+    run "UPDATE project SET reputation = 'new' WHERE reputation = 'limited'"
```

Comment:
In general, I think we should reserve raw SQL usage for cases that Sequel's API doesn't support (or are significantly clunkier when using Sequel's API):

```suggestion
    from(:project).where(reputation: "limited").update(reputation: "new")
```